test/integration/commands/xml_validate/test_reformat_input.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from lxml import etree
+
+from dsp_tools.commands.xml_validate.models.data_rdf import DataRDF
+from dsp_tools.commands.xml_validate.reformat_input import transform_into_data_deserialised
+
+
+def test_transform_into_data_deserialised(data_xml: etree._Element) -> None:
```

Comment:
I find this hard to understand. How can one easily find out where `data_xml` comes from? Actually, you only use it in this single file. So why doesn't it live here? It would be much easier to understand.

Now, it makes its way across 3 files: 

1. It is defined in xml_fixtures.py
2. Then, it is imported into conftest.py
3. Then, it magically appears here

Reply:
The reason it is in a separate file is because I plan on using it more in the future.

Reply:
But then, I would not add it to conftest.py, rather import it directly from xml_fixtures.py. Our future selves will be grateful for this.

Reply:
Yes, I prefer to do that too, but that did not work in the past. We looked at it together and I don't remember why it did not work, but we only managed to make it work with the config file. Maybe it was a linter issue... not sure.